sunpy/instr/aia/response.py

```diff
@@ -0,0 +1,462 @@
+
+        wave: array
+            wavelengths imaged by the instrument and centered around the channel wavelength
+        geoarea: float
+            the geometric area of the each channel: EUV = 83.0 cm**2, UV = 30.8 cm**2
+        elecperev: float
+            a conversion factor for electrons per electronic volts
+        ent_filter: array
+            transmission efficiency of the entrance filter
+        fp_filter: array of floats
+            transmission efficiency of the focal plane filter
+        contam: array of floats
+            the quantum efficiency of the ccd
+        cross_area: array of flats
+            cross area of the instrument
+        secondary: array of floats
+            reflectance of secondary mirror
+        primary: array
+            reflectance of the primary mirror
+        ccd: array, floats
+            the quantum efficiency of the ccd
+        """
+        assert wavelength_center in self.wavelength_centers, 'Choose one integer value from wavelength centers: ' + str(
+            self.wavelength_centers)
+
+        dict = {}
+
+        # target wavelength specifid and pull out respective properties in dataframe and apply units
+        for wavelength, values in self.dataframe.iteritems():
+            if int(wavelength) == int(wavelength_center):
+                dict['wave'] = values['wave'] * u.angstrom
+                dict['effarea'] = values['effarea'] * u.cm ** 2
+                dict['units'] = values['units']
+                dict['geoarea'] = values['geoarea'] * u.cm ** 2
+                dict['scale'] = values['scale']
+                dict['platescale'] = values['platescale'] * u.cm ** 2
+                dict['numfilters'] = values['numfilters']
+                dict['wavemin'] = values['wavemin'] * u.angstrom
+                dict['wavestep'] = values['wavestep'] * u.angstrom
+                dict['wavenumsteps'] = values['wavenumsteps'] * u.angstrom
+                dict['wavelog'] = values['wavelog'] * u.angstrom
+                dict['filtersincludemesh'] = values['filtersincludemesh']
+                dict['meshtrans'] = values['meshtrans']
+                dict['usecontam'] = values['usecontam']
+                dict['contamthick'] = values['contamthick']
+                dict['usephottoelec'] = values['usephottoelec'] * u.photon/u.electron
+                dict['elecperev'] = values['elecperev'] * (u.electron /u.ct) # u.eV?
+                dict['usephottodn'] = values['usephottodn'] * u.photon / u.ct
+                dict['elecperdn'] = values['elecperdn'] * (u.electron/u.ct)  # the ccd gain!!
+                dict['useerror'] = values['useerror']
+                dict['fp_filter'] = values['fp_filter'] * u.cm ** 2
+                dict['ent_filter'] = values['ent_filter'] * u.cm ** 2
+                dict['primary'] = values['primary'] * u.cm ** 2
+                dict['secondary'] = values['secondary'] * u.cm ** 2
+                dict['ccd'] = values['ccd'] * u.cm**2
```

Comment:
Thanks for the reference @Cadair! I'm finding it easier to clean these up after working directly with the variables. About the quantum efficiency, great! This fixes the unit issue I was having that function.   Thanks, @wtbarnes